app/views/registrations/aga_member_search.html.haml

```diff
@@ -50,6 +57,11 @@
     }
   }
 
+  const displayConnectionError = () => {
+      displaySearchingMessage(false);
+      searchResultsContainer.innerHTML = 'Cannot Connect to the API.';
```

Comment:
Let's adjust the capitalization of "Connect" here to be lowercase so it's in sentence case. "Cannot connect to the API."